client/src/components/Onboarding/OnboardingAdmin.js

```diff
@@ -1,7 +1,291 @@
-import React from 'react';
+import React, { useEffect, useCallback, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { LoadingSpinner, Message, Paginator } from '_common';
+import './OnboardingAdmin.module.scss';
+import { Button } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCheck, faTimes } from '@fortawesome/free-solid-svg-icons';
+import { v4 as uuidv4 } from 'uuid';
+import PropTypes from 'prop-types';
+import { onboardingUserPropType } from './OnboardingPropTypes';
+import OnboardingEventLogModal from './OnboardingEventLogModal';
+import OnboardingStatus from './OnboardingStatus';
+import OnboardingAdminSearchbar from './OnboardingAdminSearchbar';
 
-function OnboardingAdmin() {
-  return <div>Onboarding Admin</div>;
-}
+const OnboardingApproveActions = ({ callback }) => {
+  return (
+    <div styleName="approve-container">
+      <Button
+        className="c-button--secondary"
+        styleName="approve"
+        // eslint-disable-next-line standard/no-callback-literal
+        onClick={() => callback('staff_approve')}
+      >
+        <FontAwesomeIcon icon={faCheck} />
+        <>Approve</>
+      </Button>
+      <Button
+        className="c-button--secondary"
+        styleName="approve"
+        // eslint-disable-next-line standard/no-callback-literal
+        onClick={() => callback('staff_deny')}
+      >
+        <FontAwesomeIcon icon={faTimes} />
+        <>Deny</>
+      </Button>
+    </div>
+  );
+};
+
+OnboardingApproveActions.propTypes = {
+  callback: PropTypes.func.isRequired
+};
+
+const OnboardingResetLinks = ({ callback }) => {
+  return (
+    <div styleName="reset">
+      <Button
+        color="link"
+        styleName="action-link"
+        // eslint-disable-next-line standard/no-callback-literal
+        onClick={() => callback('reset')}
```

Comment:
Clicking this sometimes seems to raise an error in the backend:

> core_portal_workers | [2021-03-04 22:53:18,838: ERROR/ForkPoolWorker-2] Task portal.apps.onboarding.execute.execute_setup_steps[5d100d28-59a1-472e-9d02-8d52bbb174d9] raised unexpected: StepExecuteException(<portal.apps.onboarding.steps.access.RequestAccessStep object at 0x7efec0f00898>,)
core_portal_workers | Traceback (most recent call last):
core_portal_workers |   File "/opt/pysetup/.venv/lib/python3.6/site-packages/celery/app/trace.py", line 412, in trace_task
core_portal_workers |     R = retval = fun(*args, **kwargs)
core_portal_workers |   File "/opt/pysetup/.venv/lib/python3.6/site-packages/celery/app/trace.py", line 704, in __protected_call__
core_portal_workers |     return self.run(*args, **kwargs)
core_portal_workers |   File "/srv/www/portal/server/portal/apps/onboarding/execute.py", line 106, in execute_setup_steps
core_portal_workers |     raise StepExecuteException(setup_step)
core_portal_workers | portal.apps.onboarding.execute.StepExecuteException: <portal.apps.onboarding.steps.access.RequestAccessStep for sal is userwait>

Reply:
The error on the backend is normal (though I suppose more graceful onboarding halting may be in order later.) Fixed the missing status.